Example/Backpack/SwiftUI/Components/ImageGallerySlideshow/ImageGallerySlideshowExampleView.swift

```diff
@@ -49,7 +49,7 @@ struct ImageGallerySlideshowExampleView: View {
     private func image(_ number: Int) -> some View {
         Image("carousel_placeholder_\(number)")
             .resizable()
-            .aspectRatio(contentMode: .fill)
+            .aspectRatio(contentMode: .fit)
```

Comment:
In the example, setting the `.fit` mode on the image can show the effect of the design better and is more suitable for most use cases.